src/sinol_make/commands/export/__init__.py

```diff
@@ -0,0 +1,287 @@
+import os
+import glob
+import shutil
+import tarfile
+import tempfile
+import argparse
+import yaml
+
+from sinol_make import util
+from sinol_make.helpers import package_util, parsers
+from sinol_make.commands.gen import gen_util
+from sinol_make.interfaces.BaseCommand import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Class for "export" command.
+    """
+
+    def get_name(self):
+        return "export"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Create archive for oioioi upload',
+            description='Creates archive ready to upload to oioioi. '
+                        'It is possible to create ocen and dlazaw debian packages for usage in contests '
+                        'with --ocen flag.'
+        )
+
+        parser.add_argument('-o', '--output', type=str, default='export',
+                            help='output directory. Default: export')
```

Comment:
Is there any situation where an user might want to specify the directory? Maybe specifying the filename makes a bit more sense? Though, I think both are not needed.

What do you think of creating the package in the top-level directory, instead of the export/ subdirectory?